system/Validation/Validation.php

```diff
@@ -496,6 +496,10 @@ public function withRequest(RequestInterface $request): ValidationInterface
         if (strpos($request->getHeaderLine('Content-Type'), 'application/json') !== false) {
             $this->data = $request->getJSON(true);
 
+            if (! is_array($this->data)) {
+                $this->data = [];
+            }
```

Comment:
What should we do when `$this->data` is not an array?
Is it okay just to throw the value away?

Reply:
In my opinion, if we have decided to use Validation, it must be in the key-value format. If it is a special format, Validation is not applicable. Validation doesn't need to be as comprehensive as JSON schema.